### PR TITLE
reference: refine sql diagnosis.

### PR DESCRIPTION
**Pull request by reafans**
### What is changed, added or deleted? (Required)

Fix some changed place in SQL diagnosis-related system table documents.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/2522

### CODE REVIEW

**Comment by TomShawn** [comment]
Please update this PR according [this comment](https://github.com/pingcap/docs/pull/2251#discussion_r409527351).

File: reference/system-databases/cluster-hardware.md [hunk 1]
```diff
@@ -45,18 +45,18 @@ desc cluster_hardware;
...
 ```
 
 ```
 +------+-----------------+-------------+-------------+--------------------+-------+
 | TYPE | INSTANCE        | DEVICE_TYPE | DEVICE_NAME | NAME               | VALUE |
 +------+-----------------+-------------+-------------+--------------------+-------+
-| tidb | 127.0.0.1:10080 | cpu         | cpu         | cpu-logical-cores  | 8     |
-| tidb | 127.0.0.1:10080 | cpu         | cpu         | cpu-physical-cores | 4     |
-| pd   | 127.0.0.1:2379  | cpu         | cpu         | cpu-logical-cores  | 8     |
+| tidb | 0.0.0.0:4000    | cpu         | cpu         | cpu-logical-cores  | 4     |
```
Comment (crazycs520) [inline reference/system-databases/cluster-hardware.md:55]: Both cpu-logical-cores and cpu-physical-cores is 4?

File: reference/system-databases/inspection-result.md [hunk 2]
```diff
@@ -15,36 +15,38 @@ TiDB 内置了一些诊断规则，用于检测系统中的故障以及隐患。
...
-    * `current-load`：如果当前系统负载太高，会生成对应的 `warning` 诊断结果。
+    * `config`：配置一致性检测。如果同一个配置在不同实例不一致，会生成 `warning` 诊断结果。
+    * `version`：版本一致性检测。如果同一类型的实例版本不同，会生成 `critical` 诊断结果。
+    * `node-load`：如果当前系统负载太高，会生成对应的 `warning` 诊断结果。
     * `critical-error`：系统各个模块定义了严重的错误，如果某一个严重错误在对应时间段内超过阈值，会生成 `warning` 诊断结果。
     * `threshold-check`：诊断系统会对大量指标进行阈值判断，如果超过阈值会生成对应的诊断信息。
 * `ITEM`：每一个规则会对不同的项进行诊断，该字段表示对应规则下面的具体诊断项。
 * `TYPE`：诊断的实例类型，可取值为 `tidb`，`pd` 和 `tikv`。
 * `INSTANCE`：诊断的具体实例地址。
+* `STATUS_ADDRESS`：HTTP API 的服务地址。
```
Comment (crazycs520) [inline reference/system-databases/inspection-result.md:49]:
```suggestion
* `STATUS_ADDRESS`：实例的 HTTP API 服务地址。
```

File: reference/system-databases/metrics-schema.md [hunk 3]
```diff
@@ -111,7 +111,7 @@ desc select * from metrics_schema.tidb_query_duration where value is not null an
 
 从以上结果可知，在 [`2020-03-25 23:40:00`, `2020-03-25 23:42:00`] 时间范围内，每个 label 只有三个时间的值，执行计划中的 `step` 值为一分钟，这实际上是由下面两个 session 变量决定的：
 
-* `tidb_metric_query_step`：查询的分辨率步长。从 Prometheus 的 `query_range` 数据时需要指定 `start`，`end` 和 `step`，其中 `step` 会使用该变量的值。
+* `tidb_metric_query_step`：查询的分辨率步长。从 Prometheus 的 `query_range` 数据时需要指定 `start_time`，`end_time` 和 `step`，其中 `step` 会使用该变量的值。
```
Comment (crazycs520) [inline reference/system-databases/metrics-schema.md:114]:
```suggestion
* `tidb_metric_query_step`：查询的分辨率步长。从 Prometheus 的 `query_range` 接口查询数据时需要指定 `start_time`，`end_time` 和 `step`，其中 `step` 会使用该变量的值。
```

File: reference/system-databases/cluster-hardware.md (hunk 1 above)
Comment (reafans) [reply reference/system-databases/cluster-hardware.md:55]: I run it in the Virtual Machine of the Virtual Box，so it show both of them are same.

File: reference/system-databases/inspection-summary.md [hunk 4]
```diff
@@ -37,10 +39,11 @@ desc inspection_summary;
 * `INSTANCE`：监控的具体实例。
 * `METRIC_NAME`：监控表。
```
Comment (TomShawn) [inline reference/system-databases/inspection-summary.md:40]:
```suggestion
* `METRICS_NAME`：监控表。
```

File: reference/system-databases/inspection-result.md [hunk 5]
```diff
@@ -289,7 +293,7 @@ DETAILS   | the cluster has 2 different tidb versions, execute the sql to see mo
 | TiKV | region-count | pd_scheduler_store_status  | 小于 20000 | 检测各个 TiKV 节点的 Region 数量，期望单个节点的 Region 数量小于 20000 |
```
Comment (TomShawn) [inline reference/system-databases/inspection-result.md:293]: @reafans 请再确认 SQL 诊断相关系统表文档中的 `节点` 是否应该替换成 `实例`。

File: reference/system-databases/sql-diagnosis.md [hunk 6]
```diff
@@ -20,14 +20,14 @@ SQL 诊断共分三大块：
...
-* 集群负载表 [`information_schema.cluster_load`](/reference/system-databases/cluster-load.md) 用于查询集群不同节点以及不同硬件类型的负载信息。
-* 内核参数表 [`information_schema.cluster_systeminfo`](/reference/system-databases/cluster-systeminfo.md) 用于查询集群不同节点的内核配置信息。目前支持查询 sysctl 的信息。
-* 集群日志表 [`information_schema.cluster_log`](/reference/system-databases/cluster-log.md) 用于集群日志查询，通过将查询条件下推到各个节点，降低日志查询对集群的影响，性能影响小于等 grep 命令。
+* 集群负载表 [`information_schema.cluster_load`](/reference/system-databases/cluster-load.md) 用于查询集群不同实例以及不同硬件类型的负载信息。
+* 内核参数表 [`information_schema.cluster_systeminfo`](/reference/system-databases/cluster-systeminfo.md) 用于查询集群不同实例的内核配置信息。目前支持查询 sysctl 的信息。
+* 集群日志表 [`information_schema.cluster_log`](/reference/system-databases/cluster-log.md) 用于集群日志查询，通过将查询条件下推到各个实例，降低日志查询对集群的影响，性能影响小于等 grep 命令。
 
-TiDB 4.0 之前的系统表，只能查看当前节点，TiDB 4.0 实现了对应的集群表，可以在单个 TiDB 节点上拥有整个集群的全局视图。这些表目前都位于 [`information_schema`](/reference/system-databases/information-schema.md) 中，查询方式与其他 `information_schema` 系统表一致。
+TiDB 4.0 之前的系统表，只能查看当前节点信息，TiDB 4.0 实现了对应的集群表，可以在单个 TiDB 实例上拥有整个集群的全局视图。这些表目前都位于 [`information_schema`](/reference/system-databases/information-schema.md) 中，查询方式与其他 `information_schema` 系统表一致。
```
Comment (TomShawn) [inline reference/system-databases/sql-diagnosis.md:30]: 是`节点`还是`实例`呢？

File: reference/system-databases/sql-diagnosis.md [hunk 7]
```diff
@@ -20,14 +20,14 @@ SQL 诊断共分三大块：
 集群信息表将一个集群中的所有节点实例的信息都汇聚在一起，让用户仅通过一条 SQL 就能查询整个集群相关信息。
 集群信息表列表如下：
 
-* 集群拓扑表 [`information_schema.cluster_info`](/reference/system-databases/cluster-info.md) 用于获取集群当前的拓扑信息，以及各个节点的版本、版本对应的 Git Hash、各节点的启动时间、各节点的运行时间。
-* 集群配置表 [`information_schema.cluster_config`](/reference/system-databases/cluster-config.md) 用于获取集群当前所有节点的配置。对于 TiDB 4.0 之前的版本，用户必须逐个访问各个节点的 HTTP API 才能获取这些配置信息。
+* 集群拓扑表 [`information_schema.cluster_info`](/reference/system-databases/cluster-info.md) 用于获取集群当前的拓扑信息，以及各个实例的版本、版本对应的 Git Hash、各实例的启动时间、各实例的运行时间。
+* 集群配置表 [`information_schema.cluster_config`](/reference/system-databases/cluster-config.md) 用于获取集群当前所有实例的配置。对于 TiDB 4.0 之前的版本，用户必须逐个访问各个节点中实例的 HTTP API 才能获取这些配置信息。
```
Comment (TomShawn) [inline reference/system-databases/sql-diagnosis.md:24]: ditto

**Pull request merged**